setup.py

```diff
@@ -7,7 +7,7 @@
 
 setup(
     name='django-taggit-autosuggest',
-    version='0.4.1',
+    version='0.4.2',
```

Comment:
This should be a minor release instead of patch since it is dropping support for `Django 3.2` so let's update this to `0.5.0` instead.